filterNotam.py

```diff
@@ -77,4 +77,28 @@ def identify_lighting_marking_notams(notams):
 
 # This function will filter out NOTAMs that have been marked as lighting or marking NOTAMs.
 def filter_out_lighting_marking_notams(notams, marked_notams):
-    return [notam for notam in notams if notam not in marked_notams]
\ No newline at end of file
+    return [notam for notam in notams if notam not in marked_notams]
+
+def filter_classification(notams, classification):
+    marked_notams = set()
+    for notam in notams:
+        if classification in notam.classification:
```

Comment:
is the `in` operator really the right one to use? isn't `notam.classification` a string (not a list), so what you are checking for here is where the NOTAMs classification has `classification` as a substring

Reply:
Fixed in the latest commit